### DISCUSSION TITLE
Token for maintenance-window

### DISCUSSION THREAD

**Discussion by mdicss** [Dynatrace/dynatrace-configuration-as-code#330] [Archive] [answered]
Hi all
The token permission 'Configure maintenance windows' is deprecated. What is the new permission needed to define maintenance windows? I didn't find anything in the dynatrace documentation.

**Comment by tatomir146** [comment, marked as answer]
@mdicss ReadConfig and WriteConfig are necessary for this to work. `monaco` is not using the old depracated API. @CruzanCaramele can we please update this in documentation?

**Reply by mdicss** [reply]
Thanks for the answer and the update of the documentation.
It would be nice to have fine grained permissions. Our case is, that I want to have a skript to set a maintenance window and it would be good to have a token for this with no permissions to change the whole dynatrace configuration.

**Reply by tatomir146** [reply]
That's something which will IMO be significantly improved in product witch settings 2.0
